sapi/phpdbg/phpdbg.c

```diff
@@ -882,13 +882,6 @@ static ssize_t phpdbg_stdiop_write(php_stream *stream, const char *buf, size_t c
 	return PHPDBG_G(php_stdiop_write)(stream, buf, count);
 }
 
-static inline void php_sapi_phpdbg_flush(void *context)  /* {{{ */
-{
-	if (!phpdbg_active_sigsafe_mem()) {
-		fflush(PHPDBG_G(io)[PHPDBG_STDOUT].ptr);
```

Comment:
Just wondering why this is gone now?

Reply:
We no longer write buffered (FILE*) streams anywhere, this looks like a left over relic rather than necessary.

Reply:
Make sense.